model/ft_sensor.urdf.xacro

```diff
@@ -15,19 +15,23 @@
 		</joint>
 
 		<link name="${name}_${type}_base">
-			<insert_block name="nano17"/>
+			<!-- <insert_block name="nano17"/> -->
+			<insert_block name="gamma"/>
 		</link>
 
 		<joint name="${parent}_${name}_measure_joint" type="revolute">
-			<insert_block name="nano17_measure_frame"/>
+			<!-- <insert_block name="nano17_measure_frame"/> -->
+			<insert_block name="gamma_measure_frame"/>
+
 			<parent link="${name}_${type}_base"/>
 			<child link="${name}_${type}_measure"/>
-			<limit lower="0" upper="0" effort="100" velocity="100"/>
+			<limit lower="0" upper="0" effort="0" velocity="0"/>
 			<axis xyz="0 0 1" />
 		</joint>
 
 		<link name="${name}_${type}_measure">
-			<insert_block name="nano17_platform"/>
+			<!-- <insert_block name="nano17_platform"/> -->
+			<insert_block name="gamma_platform"/>
```

Comment:
Ok for now. 
We need to figure out how to do this right, i.e. pass the type as argument.